libfuse/folderlist.go

```diff
@@ -95,6 +95,20 @@ func (fl *FolderList) addToFavorite(ctx context.Context, h *libkbfs.TlfHandle) (
 	return nil
 }
 
+// Create implements the fs.NodeCreater interface for FolderList.
+func (fl *FolderList) Create(ctx context.Context, req *fuse.CreateRequest, resp *fuse.CreateResponse) (_ fs.Node, _ fs.Handle, err error) {
+	fl.fs.log.CDebugf(ctx, "FL Create")
+	defer func() { fl.reportErr(ctx, libkbfs.WriteMode, "", err) }()
+	return nil, nil, fl.fs.writeAccessError(ctx, nil, libkbfs.BuildCanonicalPath(fl.public, libkbfs.CanonicalTlfName(req.Name)))
```

Comment:
I don't think "Permission denied" is the right error, here in the folder list or in the `/keybase` operations -- it implies that there might be some user who has permission to do this thing, when in fact the operation isn't supported at all.